src/Monolog/Handler/GrabyHandler.php

```diff
@@ -45,9 +46,9 @@ public function hasRecords($level): bool
         return isset($this->recordsByLevel[$level]);
     }
 
-    protected function write(array $record): void
+    protected function write(LogRecord $record): void
```

Comment:
Looks like LogRecord is present in monolog 2.4+ https://github.com/Seldaek/monolog/blob/2.9.1/src/Monolog/LogRecord.php

Reply:
So we could do a "soft upgrade", so that the actual upgrade to monolog 3 will only be switching the dependency in composer?

Reply:
No, it is only available as a stub. The implementation would require accepting `array|LogRecord` for the time being.